pyeudiw/satosa/backend.py

```diff
@@ -450,7 +464,16 @@ def request_endpoint(self, context, *args):
         jwt = helper.sign(data)
         response = {"response": jwt}
         
-        self.db_engine.update_request_object(entity_id, nonce, state, data)
+        try:
+            self.db_engine.update_request_object(entity_id, nonce, state, data)
+        except Exception as e:
+            return JsonResponse(
```

Comment:
please add the logging call here

Reply:
Resolved in [c69e7db](https://github.com/italia/eudi-wallet-it-python/pull/89/commits/c69e7db13eddb6b3a16fc2e5d182fea4b08cfba8)